### DISCUSSION TITLE
Obtain PFAM domains used by antismash

### DISCUSSION THREAD

**Discussion by yangkky** [antismash/antismash#574] [Q&A]
Antismash runs Prodigal to detect ORFs and then hmmsearch to detect PFAM domains. These PFAM domains are then used to predict BGCs. Is it possible to see all the detected PFAM domains for each genome in addition to those that are within a BGC?

**Comment by SJShaw** [comment]
Cluster detection uses a selection of profiles from various sources, not just Pfam, and not all of Pfam.
If you want to view all Pfam domains within a genome, run with the `--fullhmmer` option, then look at the resulting `PFAM_domain` features in the Genbank output.
